.travis.yml

```diff
@@ -3,4 +3,4 @@ sudo: false
 dist: trusty
 
 script:
-  - npm run lint .
+  - npm run lint . && npm test
```

Comment:
Why would you not want to see the output of both anyway? You will still see ESLint errors if there are any, but this way you will see test failures too. (Both sections are collapsible in the webface.) The code still has to pass ESLint to pass the build.